dtcli/scripts/dt.py

```diff
@@ -806,6 +815,29 @@ def prepare_python(path_to_setup_py, **kwargs):
     )
 
 
+@utility.command()
+@click.option(
+    "--prefix",
+    default="",
+    help="Prefix added to the secret"
+)
+@click.option(
+    "--postfix",
+    default="",
+    help="Postfix added to the secret"
+)
+def acquire_secret(prefix, postfix):
+    """
+    Asks for secret and writes it to stdout with prefix and postfix.
+    """
+    secret = getpass.getpass(prompt="Secret: ")
```

Comment:
Actually the experience should be closer to a Typer prompt (without input hiding) -> see the original script for reference
We can add a red warning though that it'll be visible